halo2_proofs/examples/cost-model.rs

```diff
@@ -102,6 +100,7 @@ impl FromStr for Poly {
 
 #[derive(Debug)]
 struct Lookup {
+    #[allow(dead_code)]
```

Comment:
Is this intended?

Reply:
Clippy was complaining about it. let me check again.